test/examples/Id-Strings.jvx

```diff
@@ -0,0 +1,17 @@
+-- standard library functions
+
+let string = "%prim.string"
+
+let pair = "%pair"
+
+-- Identity contract for strings. 
+
+type storage = unit 
+
+-- Default entry point
+sig default : string -> storage -> storage
+￼let default para storage = Pair [] storage
```

Comment:
```suggestion
￼let default para storage = pair [] storage
```

do we have sugar for `[]` (cc @mariari)? this needs to translate to an empty Michelson list, if we use it like this

Reply:
We don't yet, but I can add it in 30~ minutes if you want it